tests/data/csv/pacbio.csv

```diff
@@ -1,6 +1,6 @@
 Source,Tag,Genome Size,Insert Size,Concentration,Volume
-DN814597W-A10,bc1059T,6.3,15230,13,15

```

Comment:
we might need to check where the data comes from. If it is automated qc then it might still need -. Although you may have checked ...

Reply:
I have not checked, that is a good question. I assumed we were generating it somewhere